frontend/packages/core/src/tests/tab.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import { Tab, Tabs } from "../tab";
+
+describe("Tabs component aka TabGroup", () => {
+  describe("basic rendering", () => {
+    let component;
+    beforeEach(() => {
+      component = shallow(
+        <Tabs value={1}>
+          <Tab label="meow" />
+          <Tab label="mix" />
+        </Tabs>
+      );
+    });
+
+    it("renders", () => {
+      expect(component.find(Tabs)).toBeDefined();
+    });
+
+    it("renders 2 Tabs", () => {
+      expect(component.find(Tab)).toHaveLength(2);
+    });
+
+    it("has the second tab selected (index 1)", () => {
+      expect(component.find("TabContext").prop("value")).toBe("1");
```

Comment:
Should we be expecting the mix tab to be selected and displayed?

Reply:
yes

Reply:
Are you saying I should add another test for that? I remember Josh suggested that we only have 1 `expect` per test to keep things organized

Reply:
Yeah I would add a test that it's displayed as well

Reply:
here is the debug() output for the component:
```
<div style={{...}}>
@clutch-sh/core:       <TabContext value="1">
@clutch-sh/core:         <Styled(Component) variant={[undefined]} onChange={[Function: onChangeMiddleware]}>
@clutch-sh/core:           <Tab label="meow" value="0" />
@clutch-sh/core:           <Tab label="mix" value="1" />
@clutch-sh/core:         </Styled(Component)>
@clutch-sh/core:         <Styled(WithStyles(ForwardRef(TabPanel))) value="0" />
@clutch-sh/core:         <Styled(WithStyles(ForwardRef(TabPanel))) value="1" />
@clutch-sh/core:       </TabContext>
@clutch-sh/core:     </div>
```
as you can see it doesn't give an indication that "mix" tab is selected other than the tabContext . An alternative thing I could do is maybe use the react-test renderer like Josh does in the NPS thing instead. I'll ask Josh today about it.